src/java.base/share/classes/java/lang/Object.java

```diff
@@ -36,12 +36,8 @@
  * Every class has {@code Object} as a superclass. All objects,
  * including arrays, implement the methods of this class.
  * <p>
- * Subclasses of {@code java.lang.Object} can be either {@linkplain Class#isIdentity()
- * identity classes} or {@linkplain Class#isValue value classes}.
- * The class {@code Object} itself is neither an identity class nor a value class.
- * See {@jls The Java Language Specification  8.1.1.5 identity and value Classes}.
- * An instance can be created with {@code new Object()}, those instances are
- * {@link Objects#isIdentityObject(Object) an identity object}.
+ * Subclasses of {@code java.lang.Object} can be either identity classes or value classes.
+ * See {@jls The Java Language Specification 8.1.1.5 value Classes}.
```

Comment:
This does not seem highly necessary.   If it's there, maybe worth adding the link to `Class::isIdentity` and `Class::isValue`